ambry-rest/src/test/java/com/github/ambry/rest/NettyServerTest.java

```diff
@@ -38,7 +38,7 @@ public class NettyServerTest {
   private static final RestRequestService REST_REQUEST_SERVICE =
       new MockRestRequestService(new VerifiableProperties(new Properties()), new MockRouter());
   private static final RestRequestHandler REQUEST_HANDLER = new MockRestRequestResponseHandler(REST_REQUEST_SERVICE);
-  private static final PublicAccessLogger PUBLIC_ACCESS_LOGGER = new PublicAccessLogger(new String[]{}, new String[]{});
+  private static final PublicAccessLogger PUBLIC_ACCESS_LOGGER = new PublicAccessLogger(new String[]{}, new String[]{}, false);
```

Comment:
Maybe let's add a couple of tests where `enableStructuredLogging` is set to true. Doesn't really test the log message itself, but at least we will have the confidence that the code path will not just error out, and will also be maintained in the future.

Reply:
Yeah, I modified FrontendNettyFactoryTest to include a couple of cases where where `enableStructuredLogging` is set to true. Let me know if you think that's fine. If not, I'll need some help to add tests from scratch.